tasks/sanity-label-check.yaml

```diff
@@ -23,20 +23,21 @@ spec:
       image: quay.io/redhat-appstudio/hacbs-test:stable
       workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
       script: |
-        skopeo inspect --no-tags docker://$(inputs.params.IMAGE_URL) | tee image_inspect.json
-
         echo "Running conftest using $(inputs.params.POLICY_DIR) policy, $(inputs.params.POLICY_NAMESPACE) namespace"
-        /usr/bin/conftest test --no-fail image_inspect.json \
+        /usr/bin/conftest test --no-fail ../sanity-inspect-image/image_inspect.json \
         --policy $(inputs.params.POLICY_DIR) --namespace $(inputs.params.POLICY_NAMESPACE) \
-        --output=json | tee sanity_label_check_output.json
-
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
-          '{result: "ERROR", timestamp: $date}')
+        --output=json 2> stderr.txt | tee sanity_label_check_output.json
+        
+        ERR_MSG=$(cat stderr.txt)
+        if [ ! -z "$ERR_MSG" ]; then
```

Comment:
I believe that we want to always have the `HACBS_ERROR_OUTPUT` defined. Maybe we can default to `ERR_MSG` being `unknown` if it's empty.

Reply:
> @hongweiliu17 can you also remove the [IMAGE_URL](https://github.com/redhat-appstudio/build-definitions/blob/623087defddfc87c9281a308b0674840f713a7bb/tasks/sanity-label-check.yaml#L8-L9) param from `sanity-label-check.yaml`? That parameter won't be needed anymore.

Updated. Thanks.